src/procedures/procedures.11tydata.yaml

```
@@ -0,0 +1,300 @@
+  steps:
+    - title: Unpacking the components
+      image: step-5-poe-unpacked.jpeg
+      content: |
+        <ul>
+          <li>Make sure you have all the components:</li>
+          <li>1 Raspberry Pi Compute Module 4 (CM4).</li>
+          <li>1 heat sink</li>
+          <li>2 thermal pads</li>
+          <li>2 push pins</li>
+        </ul>
+    - title: Installing the CM4
+      image: step-6-poe-install-cm4.jpeg
+      content: |
+        <ul>
+          <li>The raspberry icon needs to be upside down: check for correct orientation of the CM4.</li>
+          <li>Press the CM4 board onto the board-to-board connectors until it clicks into place.</li>
+          <li>Make sure you press on both sides (the two long edges).</li>
+          <li>Make sure the CM4 is installed correctly: it needs to run parallel to the yellow board.</li>
+        </ul>
+    - title: Placing the thermal pads onto the CM4
+      image: step-7-poe-heat-pads-placement_callouts.jpeg
+      content: |
+        <ul>
+          <li>Remove the protective cover from one side of the square thermal pad.</li>
+          <li>Stick the unprotected side onto the SOC (the silver square next to the raspberry icon).</li>
+          <li>Remove the protective cover from the top side of the thermal pad.</li>
+          <li>Repeat the steps with the rectangular thermal pad. Place it on the memory (the black rectangle next to the SOC).</li>
+        </ul>
+    - title: Placing the heat sink
+      image: step-8-install-heatsink.webp
+      content: |
+        <ul>
+          <li>Push the 2 plastic push pins into the dedicated hole on the heat sink.</li>
+          <li>Start by first aligning the right plastic push pins into the dedicated hole on the carrier board.</li>
+          <li>Then, place the heat sink onto the CM4.</li>
+        </ul>
+    - title: Securing the heat sink
+      image: step-9-push-pins-heatsink.webp
+      content: |
+        <ul>
+          <li>Take the entire board off your workplace and push the pins firmly through the carrier board.</li>
+        </ul>
+procedureInstallNvme:
+  title: Installing the NVMe drive
+  steps:
+    - title: Preparation
+      image: step-8.png
+      content: |
+        <ul>
+          <li>Unscrew the stand-off nut.</li>
+        </ul>
+    - title: Inserting the drive
+      image: step-9.png
+      content: |
+        <ul>
+          <li class="warning">Notice! Do not touch the gold connector pins.</li>
+          <li>Carefully hold the NMVe M.2 SSD by the sides.</li>
+          <li>Align the notches in the SSD with the ridges in the slot.</li>
+          <li>Then, insert at a 30-degree angle.</li>
+          <li>Do not force the connection.</li>
+        </ul>
+    - title: Installing the stand-off nut
+      image: step-10.png
+      content: |
+        <ul>
+          <li>Slide the stand-off nut into the dent at the end of the SSD drive.</li>
+          <li>Make sure the thinner end of the nut is on top.</li>
+        </ul>
```

```suggestion
      steps:
        - Slide the stand-off nut into the dent at the end of the SSD drive.
        - Make sure the thinner end of the nut is on top.
```
Since you are already changing it, wouldn't this make more sense as well?
And then just iterate over them in the template

---

There is one instance which has a paragraph before the unordered list, but other than that, its always an unordered list. I guess we could allow to have a content (the paragraph) and steps afterwards.